umbraco-cloud/security.md

```diff
@@ -150,14 +152,11 @@ You can read more about the HTTP request headers coming from Cloudflare in the [
   <action type="CustomResponse" statusCode="403" />
 </rule>
 ```
+{% endtab %}
+{% tab title="Legacy (non-reverse proxy)" %}
+**Projects from before December 8th, 2020 (Non-Reverse Proxy, eg. non-Cloudflare)**
```

Comment:
Tried to shorten this heading as well. Not sure the "create date" part if necessary, as we could mention that in the in the tab instead 🤔 

```suggestion
{% endtab %}

{% tab title="Legacy (non-reverse proxy)" %}
**Non-Reverse Proxy (create date < 2021)**
```

Reply:
Changed it a little bit as to improve readability